src/rgw/rgw_rados.cc

```diff
@@ -3764,6 +3764,22 @@ int RGWRados::init_rados()
     }
   }
 
+  {
+    map<string,string> metadata;
+    metadata["pid"] = stringify(getpid());
+    char hostname[1024] = {0};
+    gethostname(hostname, sizeof(hostname));
+    metadata["host"] = hostname;
+    metadata["num_handles"] = handles.size();
+    metadata["zonegroup_id"] = zonegroup_id;
+    metadata["zone_name"] = zone_name;
+    string name = g_conf->name.get_id();
+    if (name.find("rgw.") == 0) {
+      name = name.substr(4);
```

Comment:
@dillaman wouldn't it be usefult to have some grouping, e.g.,
"rgw" {
   "radosw" {
     ... }
   "nfs" {
   }
  ...
}